default/resident/channels.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "name"    : "activation",
+        "type"    : "EMAIL",
+        "address" : "ace_de@oracle.com"
```

Comment:
This should be sanitized, requiring the user to enter something before it would be usable.

Reply:
Activation only - resolved